src/pages/Messages.jsx

```diff
@@ -22,51 +20,29 @@ import { MessageButtonGroup, MessageFolder } from '../components/Messages';
  */
 const Messages = () => {
   localStorage.setItem('restorePath', '/messages');
-  const { podUrl } = useContext(SignedInUserContext);
-  const { session } = useSession();
+  const { updateMessageCountState } = useContext(MessageContext);
   const {
```

Comment:
The two hooks in `Message.jsx` will be moved into their own pages in a follow-up PR, one for "Inbox", one for "Outbox".